src/ratings-and-reviews/index.jsx

```diff
@@ -19,35 +18,21 @@ const reviewsTotal = (reviews) => {
   return total;
 };
 
-/** @param {Object} reviews*/
-const reviewsAverage = (reviews) => {
-  let total = 0;
-  let count = 0;
-  for (let i in reviews) {
-    total += i * reviews[i];
-    count += reviews[i];
-  }
-  return parseFloat(total / count).toFixed(1);
-};
-
-let averageRating = reviewsAverage(reviewMetaData.ratings);
-
-
-const RatingsAndReviews = () => (
+const RatingsAndReviews = (/* { reviews, metadata } */) => (
```

Comment:
@ajpsyk If at some point you want to start using API data, simply uncomment these props so you get:
```js
const RatingsAndReviews = ({ reviews, metadata }) => (
```
At that point you can also remove the `./sampleData.js` import.